cdk/resources/create_config_lambda/requirements.txt

```diff
@@ -0,0 +1,9 @@
+boto3==1.17.97
```

Comment:
These requirements can get outdated fast. If not Pipfile, can we use a `requirements.in` file ([example](https://github.com/Netflix/consoleme/blob/master/requirements.in)), and use pip-tools to generate the pinned dependencies? ([sample command](https://github.com/Netflix/consoleme/blob/master/Makefile#L109))

Reply:
I'm trying to avoid using another tool for dependencies resolving. Maybe ill try to generate the requirements file using `pipfile lock`

Reply:
Used `pipreqs` to automatically create `requirements.txt` file from the `create_config_lambda` imports.